### PR TITLE
optimize: no longer substring the message in rpc transmission

### PR DESCRIPTION
**Pull request by l81893521**
### Ⅰ. Describe what this PR did
User Feedback: Misunderstand the message when transaction error occurred in TC. Because if the message longer than 128 byte, and TC would substring the message before rpc transmission.

no longer substring the message in rpc transmission and show the hold message especially in transaction error.

### CODE REVIEW

**Review by wangliang181230 — APPROVED** [review]
LGTM

**Review by lightClouds917 — APPROVED** [review]
LGTM

**Comment by jsbxyyx** [comment]
We recommend keeping the original situation. for example: adding `go to seata-server view log` in message.

**Review by a364176773 — APPROVED** [review]
LGTM

File: serializer/seata-serializer-seata/src/main/java/io/seata/serializer/seata/protocol/AbstractResultMessageCodec.java [hunk 1]
```diff
@@ -45,13 +43,7 @@
...
             if (StringUtils.isNotEmpty(resultMsg)) {
-                String msg;
-                if (resultMsg.length() > MAX_ERR_MSG_LEN) {
-                    msg = resultMsg.substring(0, MAX_ERR_MSG_LEN) + "...";
-                } else {
-                    msg = resultMsg;
-                }
-                byte[] bs = msg.getBytes(UTF8);
+                byte[] bs = resultMsg.getBytes(UTF8);
                 out.writeShort((short)bs.length);
```
Comment (slievrly) [inline serializer/seata-serializer-seata/src/main/java/io/seata/serializer/seata/protocol/AbstractResultMessageCodec.java:47]: short  should need to be less  2^15-1

Comment (l81893521) [reply serializer/seata-serializer-seata/src/main/java/io/seata/serializer/seata/protocol/AbstractResultMessageCodec.java]: fixed.

**Review by slievrly — APPROVED** [review]
LGTM

**Pull request merged**